plugins/woocommerce/includes/blocks/class-wc-brands-block-template-utils-duplicated.php

```diff
@@ -361,7 +361,8 @@ public static function should_use_blockified_product_grid_templates() {
 		$use_blockified_templates = wc_string_to_bool( get_option( Options::WC_BLOCK_USE_BLOCKIFIED_PRODUCT_GRID_BLOCK_AS_TEMPLATE ) );
 
 		if ( false === $use_blockified_templates ) {
-			return function_exists( 'wc_current_theme_is_fse_theme' ) && wc_current_theme_is_fse_theme();
+			return function_exists( 'wc_current_theme_is_fse_theme' ) && wp_is_block_theme()
```

Comment:
This will break as we remove `wc_current_theme_is_fse_theme`. You'll need to remove the `function_exists` check